### ISSUE TITLE
need e2e test for etcd backup and restore

### ISSUE THREAD

**Issue by jim-minter** [openshift/openshift-azure#379]
@mjudeikis ;-)

**Comment by mjudeikis** [comment]
These ones will need to `go logic` compatible test framework.  We will need to create a cluster, and check blob to be available.

**Comment by mjudeikis** [comment]
This is blocked by:
https://github.com/openshift/openshift-azure/issues/482 (https://github.com/openshift/openshift-azure/pull/483) 
https://github.com/openshift/openshift-azure/pull/509
https://github.com/openshift/openshift-azure/pull/492

**Comment by kargakis** [comment]
Closing in favor of https://jira.coreos.com/browse/AZURE-228
